gdx/src/com/badlogic/gdx/graphics/g2d/Sprite.java

```diff
@@ -529,7 +529,17 @@ public float getX () {
 	public float getY () {
 		return y;
 	}
-
+	
+	/** @return the center x of the sprite, not accounting for scale. */

```

Comment:
Let's account for scale and rotation as well, otherwise this is misleading unless you read the documentation (and a lot less useful).